### ISSUE TITLE
Example Module Violates Naming Conventions

### ISSUE THREAD

**Issue by sshymko** [TexanHogman/m2-minimal#1]
Module name `TexanHogman_M2-Minimal` does not comply with the module naming conventions of Magento 2. In particular, config will not pass the XSD validation against [module.xsd](https://github.com/magento/magento2/blob/develop/lib/internal/Magento/Framework/Module/etc/module.xsd#L76) because of the dash character.

**Comment by TexanHogman** [comment]
Yes, this was a very early personal example put together; samples will rollout https://github.com/magento/magento2-samples.  I will rename/remove to avoid confusion

**Comment by TexanHogman** [comment]
readme,md updated to indicate this is not an example.
